src/nemo-connectivity/connectionhelper.cpp

```diff
@@ -134,6 +134,15 @@ ConnectionHelper::Status ConnectionHelper::status() const
     return m_status;
 }
 
+void ConnectionHelper::setSelectorVisible(bool selectorVisible)
+{
+    if (m_selectorVisible != selectorVisible)
+    {
+        m_selectorVisible = selectorVisible;
+        emit selectorVisibleChanged();
```

Comment:
```suggestion
        emit selectorVisibleChanged(m_selectorVisible);
```

Reply:
Passing the current value as a signal parameter is generally pretty useless because it can't be trusted. If one of the signal handlers can change the value, that would cause the signal parameter to go out of sync with the reality for the subsequent handlers. Or maybe even the second change could arrive before the first one, although I'm not 100% about that. In any case such parameters better be ignored and the current value queried directly from the object.

In this particular case it doesn't really matter since the value is readonly, i.e. such parameter would be useless but harmless.

Reply:
It can be sometimes nice in the c++ side but not much need in qml. Suggest not changing anything here.